2023-03_sprint/sprint_parties/sponsors.md

```diff
@@ -0,0 +1,31 @@
+# Data Umbrella Sponsors
+::::{grid} 1 1 2 2
+:::{grid-item-card}
+:link: https://opensource.google/
+:shadow: none
+:class-card: border-0
+
+![Google](../../_static/sponsors/google_color.png)
+
+We bring all the value of open source to Google and all the resources of Google to open source.
+:::
+:::{grid-item-card}
+<!-- :link: https://www.pymc-labs.io/
+:shadow: none
+:class-card: border-0
+
+![PyMC Labs](../../_static/sponsors/4-pymc-labs-transp-black.png)
+
+The Bayesian Consultancy. We bring Bayesian methods into wide practice. 
+:::
+:::{grid-item-card} -->
+:link: https://chanzuckerberg.com/science/programs-resources/open-science/communitiesofpractice/data-umbrella/
+:shadow: none
+:class-card: border-0
+
+![CZI](../../_static/sponsors/czi-logo.jpg)
+
+We help solve some of society’s toughest challenges — from eradicating disease and improving education, to addressing the needs of our local communities. Our mission is to build a more inclusive, just, and healthy future for everyone.
```

Comment:
it is an issue with the comments. Here, we start a grid element, comment the metadata and its content, then comment a closing and opening of new grid element but not it's metadata. However, unlike with content where having comments around works, the metadata needs to be right in the next line after starting a directive, so rendering breaks. Again, I think it would be easier to read and to see issues if we deleted elements instead of commenting them out